draft-ietf-rats-architecture.md

```diff
@@ -346,6 +346,12 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
```

Comment:
Attesting environments may include...
<the reason for adding 'may' is because not all instances of environments listed are capable of securely collection claims for a given Target environment. Attesting environments (typically?) are built with the expected target environment in mind. Maybe we need more clarification in the text?>